nsv13/code/game/objects/structures/custom_closets.dm

```diff
@@ -124,3 +124,164 @@
 	icon = 'nsv13/icons/obj/custom_closets.dmi'
 	icon_state = "definitelynotpuce"
 	anchored = FALSE
+
+//solgov lockers
+/obj/structure/closet/secure_closet/captains/solgov
+	name = "\proper captain's locker"
+	req_access = list(ACCESS_CAPTAIN)
+	icon_state = "cap"
+
+/obj/structure/closet/secure_closet/captains/solgov/PopulateContents()
+	..()
```

Comment:
calling ..() here will also fill it with all of the items in the parent locker type so they get double items.
Ditto for every other locker
```suggestion
```